subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_sim_nrf54l.h

```diff
@@ -30,209 +30,203 @@
-#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S2_DEFAULT_NO_HW_TIFS_NS 129700
+#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S2_DEFAULT_NO_HW_TIFS_NS 130000
 #define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S2_DEFAULT_NO_HW_TIFS_US \
 	HAL_RADIO_NS2US_CEIL( \
 		HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S2_DEFAULT_NO_HW_TIFS_NS)
 
 /* RXEN->RXIDLE + RXIDLE->RX (with fast Radio ramp-up mode)
  * in microseconds for LE Coded PHY [S8].
  */
-#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_FAST_NS 40300 /* 40.1 + 0.2 */
+#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_FAST_NS 40000
 #define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_FAST_US \
 	HAL_RADIO_NS2US_CEIL(HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_FAST_NS)
 
 /* RXEN->RXIDLE + RXIDLE->RX (with default Radio ramp-up mode)
  * in microseconds for LE Coded PHY [S8].
  */
-#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NS 120200 /* 120.0 + 0.2 */
+#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NS 120000
 #define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_US \
 	HAL_RADIO_NS2US_CEIL(HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NS)
 
 /* RXEN->RXIDLE + RXIDLE->RX (with default Radio ramp-up mode and
  * no HW TIFS auto-switch) in microseconds for LE Coded PHY [S8].
  */
 /* 129.5 + 0.2 */
-#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NO_HW_TIFS_NS 129700
+#define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NO_HW_TIFS_NS 130000
 #define HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NO_HW_TIFS_US \
 	HAL_RADIO_NS2US_CEIL( \
 		HAL_RADIO_NRF54LX_RXEN_RXIDLE_RX_S8_DEFAULT_NO_HW_TIFS_NS)
 
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_1M_US  1 /* ceil(0.6) */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_1M_NS  600 /* 0.6 */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_2M_US  1 /* ceil(0.6) */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_2M_NS  600 /* 0.6 */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_S2_US  1 /* ceil(0.6) */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_S2_NS  600 /* 0.6 */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_S8_US  1 /* ceil(0.6) */
-#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_S8_NS  600 /* 0.6 */
-
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_1M_US  10 /* ceil(9.4) */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_1M_NS  9400 /* 9.4 */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_2M_US  5 /* ceil(5.0) */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_2M_NS  5000 /* 5.0 */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S2_US  25 /* ceil(19.6) */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S2_NS  24600 /* 19.6 */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S8_US  30 /* ceil(29.6) */
-#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S8_NS  29600 /* 29.6 */
+#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_US  1
+#define HAL_RADIO_NRF54LX_TX_CHAIN_DELAY_NS  1000
+
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_1M_US  9
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_1M_NS  9000
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_2M_US  5
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_2M_NS  5000
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S2_US  30
+#define HAL_RADIO_NRF54LX_RX_CHAIN_DELAY_S2_NS  30000
```

Comment:
@cvinayak , did you measure this value?
@Tschet1 has measured this value to be 26600 us

Reply:
@rugeGerritsen these are the values for the simulated target :), they are just what the simulated nrf54L15 does (which are the same timings as for the simulated 52833)

Reply:
Should we strive to make them match the real HW? 52833 and 54L behave slightly differently

Reply:
In the longer term yes. It has not been something people complained about yet so I have not prioritized it. Though I have a ticket to at least fix the different chain delay in coded phy.

Reply:
I guess the most important thing is that these constants match what the controller assumes they are. I guess this works nicely with the Zephyr controller but may not work with others (SDC?)

Reply:
So the constants in the HW models started as what the spec said for the 52832 7 or 8 years ago :),
and they never changed.
For coded Phy I just never got to add the proper difference in chain delay between both coding rates (Therefore the ticket being still there).
So yep.. that may be a bit off for the SDC or other controllers if they assume timings that are too much off from the HW models. But nobody complained so far so I hope it is an indication that it is not too bad.

Reply:
> @cvinayak , did you measure this value? @Tschet1 has measured this value to be 26600 us

The h/w value for nRF52x and nRF53x where contributed as initial contribution and never changed since then, I believe it was obtained from some internal documentations (for which on the contributor knew the document!). 

For the nRF54L, I just made a copy of the ones from nRF53. 

That said, there seems to be a regression in nRF53 when it comes to reception of S2 and tIFS switching in the Zephyr Controller. I will bisect and send a PR if I get to the bottom of it.

Reply:
Thanks @cvinayak that is good to know. But note that this PR is really independent of it. Meaning, independently of the real HW timings, the simulator timings are really what I linked above :)

Reply:
Regression in Zephyr Controller should be fixed by https://github.com/zephyrproject-rtos/zephyr/pull/84265

Tested successfully the IOP using `tests/bluetooth/shell` application with `CONFIG_BT_CTLR_PHY_CODED=y` between nrf52840dk as central and nrf54l15dk as peripheral using `bt phy-update 4 2 s2` and `bt phy-update 2 4 s2` commands, respectively.

Uses the last measured S2 Rx delay timing measurement from 2019 on nRF52840 back then for all nrf52/nrf53/nrf54l: a1cbbbe9bde71af072969e2f08070579bae164fd

Timing changes required for nRF54L will be addressed in separate PR, if required.